docs/docs/03-tools/04-credentials.md

```diff
@@ -50,6 +49,30 @@ credentials: credential-tool-1.gpt, credential-tool-2.gpt
 (tool stuff here)
 ```
 
+## Credential Tool Arguments
+
+A credential tool may define arguments. Here is an example:
+
+```yaml
+name: my-credential-tool
+args: env: the environment variable to set
+args: val: the value to set it to
+
+#!/usr/bin/env bash
+
+echo "{\"env\":{\"$ENV\":\"$VAL\"}}"
+```
+
+When you reference this credential tool in another file, you can use syntax like this to set both arguments:
+
+```yaml
+credential: my-credential-tool.gpt with MY_ENV_VAR as env and "my value" as val
```

Comment:
@g-linville WDYT about making the operand order for setting arguments with `as` match the order for aliases?

e.g. 

```suggestion
credential: my-credential-tool.gpt as myAlias with env as MY_ENV_VAR and val as "my value"
```

i.e. 

`{replacee} as {replacement}`

Reply:
Discussed in Slack. We're leaving this as is for now